modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -62,30 +62,30 @@ When it comes to connectors, each connector defines its error type hierarchy con
 
 Below is listed a full list of Error Types:
 
-* `ANY`: Error type that matches all error types that may occur in an Flow, that can be handled. This does not include errors that may occur on the source. 
-** `TRANSFORMATION`: indicates an error ocurred while transforming a value.
+* `ANY`: Error type that matches all error types that may occur in an Flow and can be handled. This does not include errors that may occur on the source. 
+** `TRANSFORMATION`: indicates an error ocurred while transforming a value. This involves Mule Runtime transformation and not DataWeave transformations. For example, using the Expression Transformer. 
 ** `EXPRESSION`: indicates an error ocurred while evaluating an expression.
 ** `VALIDATION`: indicates a validation error occured.
 *** `DUPLICATE_MESSAGE`: indicates a validation error regarding a message being processed twice. For example, using the xref:idempotent-message-validator.adoc[Idempotent message validator].
 ** `REDELIVERY_EXHAUSTED`: indicates that max attemps to reprocess a message from a source has been exhausted.
-** `CONNECTIVITY`: indicates that there was a problem establishing a connection. For example, any connector. 
-*** `RETRY_EXHAUSTED`: indicates that retries of a certain execution block has been exhausted. For example, for a given extension, or using xref:until-successful-scope.adoc[Until Successful Scope].
+** `CONNECTIVITY`: indicates that there was a problem establishing a connection. This could occur while using a connector, for example a HTTP requester. 
```

Comment:
`an HTTP`

Reply:
```suggestion
** `CONNECTIVITY`: indicates that there was a problem establishing a connection. This could occur while using a connector, for example an HTTP requester. 
```